santa/santa.go

```diff
@@ -27,11 +27,11 @@ type Rule struct {
 // Preflight representssync response sent to a Santa client by the sync server.
 type Preflight struct {
 	ClientMode                    ClientMode `json:"client_mode" toml:"client_mode"`
-	BlacklistRegex                string     `json:"blacklist_regex" toml:"blacklist_regex"`
-	WhitelistRegex                string     `json:"whitelist_regex" toml:"whitelist_regex"`
+	BlocklistRegex                string     `json:"blocklist_regex" toml:"blocklist_regex"`
+	AllowlistRegex                string     `json:"allowlist_regex" toml:"allowlist_regex"`
 	BatchSize                     int        `json:"batch_size" toml:"batch_size"`
 	EnableBundles                 bool       `json:"enable_bundles" toml:"enable_bundles"`
-	EnabledTransitiveWhitelisting bool       `json:"enabled_transitive_whitelisting" toml:"enabled_transitive_whitelisting"`
+	EnabledTransitiveallowlisting bool       `json:"enabled_transitive_allowlisting" toml:"enabled_transitive_allowlisting"`
```

Comment:
The entries in this struct are `MixedCaps` so it should be `EnabledTransitiveAllowlisting`.

Reply:
Yup this was an accident. Should be fixed now.